system_test/run.py

```diff
@@ -98,38 +99,38 @@ def setUpModule():
     # Clear our topic from Kafka
     try:
         subprocess.check_output([os.path.join(this_dir, 'standalone-kafka/kafka/bin/kafka-topics.sh'),
-                               '--zookeeper', 'localhost:2181', '--delete', '--topic', 'system-test']);
+                                 '--bootstrap-server', 'localhost:9092', '--delete', '--topic', 'system-test']);
     except subprocess.CalledProcessError as e:
         # if the complaint is that the topic doesn't exist then ignore it, otherwise fail loudly
-        if e.output.find("Topic system-test does not exist on ZK path") < 0:
+        if e.output.find("Topic system-test does not exist as expected") < 0:
             raise e
 
     # Recreate fresh
     output = subprocess.check_output([os.path.join(this_dir, 'standalone-kafka/kafka/bin/kafka-topics.sh'),
-                                    '--zookeeper', 'localhost:2181', '--create', '--topic', 'system-test',
+                                    '--bootstrap-server', 'localhost:9092', '--create', '--topic', 'system-test',
                                     '--partitions', '1', '--replication-factor', '1']);
-    if output != "Created topic \"system-test\".\n":
+    if str(output, 'UTF-8') != "Created topic system-test.\n":
         raise RuntimeError("Failed to create test topic:\n{}".format(output))
 
     # Run fakeS3
-    print "Starting FakeS3..."
-    g_fakes3_proc = subprocess.Popen(['fakes3', '-p', '9090', '-r', fakes3_data_path],
-                                    stdout=subprocess.PIPE,
-                                    stderr=subprocess.STDOUT);
+#     print("Starting FakeS3...")
+#     g_fakes3_proc = subprocess.Popen(['fakes3', '-p', '9090', '-r', fakes3_data_path],
+#                                     stdout=subprocess.PIPE,
+#                                     stderr=subprocess.STDOUT);
```

Comment:
Should this code be removed?

Reply:
Deletion of the entire `system_test` directory will occur as a part of https://sugarcrm.atlassian.net/browse/CXP-2417